README.md

```diff
@@ -1,13 +1,13 @@
-# website
+# Website
```

Comment:
Why jump from `#` to `###`

Always have linear increments, so make it an h2 please (`##`)

Reply:
Website is h1, CSS Structure should be h2

Reply:
Okay. Done!